bedrock/firefox/templates/firefox/accounts.html

```diff
@@ -1,346 +0,0 @@
-{# This Source Code Form is subject to the terms of the Mozilla Public
-  # License, v. 2.0. If a copy of the MPL was not distributed with this
-  # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
-
-{% from "macros-protocol.html" import feature_card with context %}
-{% from "macros.html" import fxa_email_form, fxa_link_fragment, fxa_cta_link, google_play_button with context %}
-
-{% extends "firefox/base/base-protocol.html" %}
-
-{% set _entrypoint = 'mozilla.org-firefox-accounts' %}
-
-{% block page_title %}{{ _('Get a Firefox Account – Keep your data private, safe and synced') }}{% endblock %}
-{% block page_desc %}
-{% if l10n_has_tag('firefox_send_20190420') %}
-  {{ _('Securely sync your passwords, bookmarks and tabs across all your devices. Get a Firefox Account now – One login – Power and privacy everywhere.') }}
-{% else %}
-  {{ _('Securely sync your passwords, bookmarks and tabs across all your devices. Get a Firefox Account now – One log-in – Power and privacy everywhere.') }}
-{% endif %}
-{% endblock %}
-
-{% block page_css %}
- {{ css_bundle('firefox_accounts') }}
```

Comment:
Please remove this bundle in [static-bundles.json](https://github.com/mozilla/bedrock/blob/master/media/static-bundles.json#L767-L773)

You can also delete the `css/firefox/accounts.scss` file.